extensions/objects/templates/filepath-editor.component.ts

```diff
@@ -143,7 +143,10 @@ export class FilepathEditorComponent implements OnInit, OnChanges {
   entityId: string;
   entityType: string;
   // Check the note before imports and after fileoverview.
-  private imgData;
+  private imgData: {
+    safeValue?: SafeResourceUrl | string;
+    unsafeValue?: string;
+  } = {};
```

Comment:
Why is this set to `{}`?

Reply:
Because I am not setting imgData anywhere else in the component. I am directly setting imgData.safeValue. If I don't initialise is with `{}` I can't set safeValue of undefined (imgData.safeValue would sort of become undefnied.safeValue). That would give an error.

Reply:
Ah, I see, thanks.